src/mote/templates/mainpage.html

```diff
@@ -33,11 +33,13 @@
         <script src="{{ url_for('static', filename='jscn/fragment.js') }}"></script>
         <script src="{{ url_for('static', filename='jscn/easy.qrcode.min.js') }}"></script>
         <script src="{{ url_for('static', filename='jscn/socket.io.js') }}"></script>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/fullcalendar@5.10.1/main.css">
+    <script src="https://cdn.jsdelivr.net/npm/fullcalendar@5.10.1/main.js"></script>
```

Comment:
Please download them into the `/static/` subdirectory and load them up from there.

Also, rename `main.css` and `main.js` aptly to be representative of what they do - say, call them `fullcalendar.css` and `fullcalendar.js` (or something like that).